### PR TITLE
Fixed broken links in interop docs

### PR DESCRIPTION
**Pull request by rpetrusha**
# Fixed broken links in interop docs

### CODE REVIEW

File: docs/framework/interop/blittable-and-non-blittable-types.md [hunk 1]
```diff
@@ -55,26 +55,26 @@ Most data types have a common representation in both managed and unmanaged memor
...
   
  Some managed data types require a different representation in an unmanaged environment. These non-blittable data types must be converted into a form that can be marshaled. For example, managed strings are non-blittable types because they must be converted into string objects before they can be marshaled.  
   
- The following table lists non-blittable types from the <xref:System> namespace. [Delegates](http://msdn.microsoft.com/library/d176ee76-f982-494b-b03d-92e4118896e2), which are data structures that refer to a static method or to a class instance, are also non-blittable.  
+ The following table lists non-blittable types from the <xref:System> namespace. [Delegates](https://msdn.microsoft.com/en-us/library/d176ee76-f982-494b-b03d-92e4118896e2(v=vs.100)), which are data structures that refer to a static method or to a class instance, are also non-blittable.
```
Comment (mairaw) [inline docs/framework/interop/blittable-and-non-blittable-types.md]: no locale on links

**Review by mairaw — APPROVED** [review]
Changes look good but links should be without locales

File: docs/framework/interop/blittable-and-non-blittable-types.md (hunk 1 above)
Comment (rpetrusha) [reply docs/framework/interop/blittable-and-non-blittable-types.md]: Argh... How could I forget, @mairaw?

**Comment by mairaw** [comment]
The warnings all seem unrelated to your change. I'd merge.

**Pull request merged**